components/telekonseling/__tests__/telekonseling-link-section.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import { TelecounselingSection } from "../telekonseling-link-section";
+import telekonseling from "~/lib/content/telekonseling";
+
+const { supports } = telekonseling;
+const [support] = supports;
```

Comment:
Please avoid testing the actual content, because when the actual content gets larger, the test would unnecessarily run longer.

That's why it's important to mock the `content/*` modules, as you can see in this mock of [content/donasi.ts](https://github.com/kawalcovid19/wargabantuwarga.com/blob/5d7211217423847bc8e6e50e518015700cd24aa1/lib/content/__mocks__/donasi.ts) module.

Reply:
Based on my comment [above](https://github.com/kawalcovid19/wargabantuwarga.com/pull/722#discussion_r690891789), I got stucked when trying to solve this request. The ```telekonseling-link-section``` was made because ```education-link-section``` 'enlighten' me. That's why I didn't write mock lines like I did on ```telekonseling``` test when I don't actually have that mock file.

Did I confused you with what I was saying, Mas @zainfathoni? My grammar is as bad as my code :sweat_smile:

When I checked where the ```telekonseling-link-section``` went, it connected with ```telekonseling``` which already have the mock file it needs. Is it means that they serve the same data? So, do I have to make separate mock file for ```telekonseling-link-section``` or I don't? I'm glad if I don't but when I do, I still confused about the directory to put the mock file because the ```.tsx``` files are actually on different directory.

I was meant to help on an issue but I guess I'm giving more issue to think of. Sorry for the inconvenience :sob:

Reply:
What you need to do is simply mock the `telekonseling` data explicitly like what I just did in 8b94a569bdb4b1d6e2eaadd028f8181a1dddd166 😁 .